webapp/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "description": "This plugin serves as a starting point for writing a Mattermost plugin.",
   "main": "src/index.js",
+  "private": true,
```

Comment:
Same question here :wink:

Reply:
This should go to the starter template. It is only to remove some warnings during the building, but make sense to add it.